keyboards/boardwalk/keymaps/stecha/keymap.c

```diff
@@ -0,0 +1,61 @@
+/*
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+// Layer shorthand
+#define _QW 0
+#define _NUM 1
+#define _SYM 2
```

Comment:
```suggestion
enum layers {
    _QW,
    _NUM,
    _SYM,
};
```

Reply:
Thanks for all the suggestions, drashna! Will make the required changes and look into userspaces as well.